Quick/DSL/World+DSL.swift

```diff
@@ -17,8 +17,15 @@ extension World {
     }
 
     internal func describe(description: String, flags: FilterFlags, closure: () -> ()) {
+        guard currentExampleGroup != nil else {
+            NSException(name: NSInternalInconsistencyException,
+                reason:"Example group was not created by its parent QuickSpec spec. Check that describe() or context() was used in QuickSpec.spec() and not a more general context (i.e. an XCTestCase test)",
+                userInfo: nil).raise()
+            return
```

Comment:
Apple should make `-[NSException raise]` a `noreturn` func so this extra `return` isn't necessary :disappointed: